sealer panic when building cluster image

### What happen?

_No response_

### Relevant log output?

```Shell
Pulling image: docker.io/bitnami/mysql:8.0.29-debian-10-r23
Pulling image: docker.io/bitnami/bitnami-shell :10-debian-10-r431
Pulling image: docker.io/bitnami/mysqld-exporter:0.14.0-debian-10-r75
Pulling image: docker.io/bitnami/redis:6.2.7-debian-10-r23
Pulling image: docker.io/bitnami/redis-sentinel :6.2.7-debian-10-r23
Pulling image: docker.io/bitnami/redis-exporter: 1.37.0-debian-10-r63
Pulling image: docker.io/bitnami/bitnami-shell: 10-debian-10-r432
8904a6c7c467: Download complete
e6a50b627bcb: Download complete
dbb9bc619c4f: Download complete
213ec9aee27d: Download complete
c672fbd02587: Download complete
2232dadbeacf: Download complete
6a2a8eddaea3: Download complete
76762d319f3: Download complete
ef053557858e: Download complete
92c6c9bd0d84: Downloading [====>]200.5MB/204.5MB
e992a8800eca: Download complete
133767f7d45f: Download complete
c3461af354fa: Download complete
5ee56c8db25f: Download complete
db26075ba035: Download complete
panic: runtime error: slice bounds out of range [:19] with length 0
goroutine 274 [running] :
github.com/sealerio/sealer/pkg/image/save.(*DefaultImageSaver).saveBlobs.funcZ()
/Users/huaiyou.cyz/go/src/github.com/VinceCui/sealer/pkg/image/save/save.go:311+0xb76
golang.org/x/sync/errgroup.(*Group).Go. func1()
/Users/huaiyou.cyz/go/src/github.com/VinceCui/sealer/vendor/golang.org/x/sync/errgroup/errgroup.go:57+0x67
created by golang.org/x/sync/errgroup.(*Group).Go
/Users/huaiyou.cyz/go/src/github.com/VinceCui/sealer/vendor/golang.org/x/sync/errgroup/errgroup.go:54+0x92
```


### What you expected to happen?

build success

### How to reproduce it (as minimally and precisely as possible)?

_No response_

### Anything else we need to know?

_No response_

### What is the version of Sealer you using?

_No response_

### What is your OS environment?

_No response_

### What is the Kernel version?

_No response_

### Other environment you want to tell us?

- Cloud provider or hardware configuration:
- Install tools:
- Others:

--- a/pkg/image/save/save.go
+++ b/pkg/image/save/save.go
@@ -307,6 +307,9 @@ func (is *DefaultImageSaver) saveBlobs(imageDigests []digest.Digest, repo distri
 				<-numCh
 			}()
 
+			if len(string(tmpBlob)) < 19 {
+				return nil
+			}
 			simpleDgst := string(tmpBlob)[7:19]
 
 			_, err = blobStore.Stat(is.ctx, tmpBlob)

